src/terraform-command.js

```diff
@@ -173,14 +186,88 @@ class TerraformCommand extends AbstractCommand {
    */
   getIncludesRegex() {
     return this.getOption('include-regex').map(it => new RegExp(it));
-  }  
-  
+  }
+
   /**
-  * @returns {RegExp[]}
-  */
+   * @returns {RegExp[]}
+   */
   getExcludesRegex() {
     return this.getOption('exclude-regex').map(it => new RegExp(it));
   }
+
+  /**
+   * Get Project CI mapping
+   * @return {Object}
+   */
+  getProjectCi() {
+    return this._configLoader.getProjectCi();
+  }
+
+  /**
+   * @return {String[]}
+   */
+  getGitDiff() {
+    const commits = this.getOption('git-diff');
+
+    if (!commits.length) {
```

Comment:
this is covered in `.addOption('git-diff', 'g', 'To be updated', Array, [])`, am I wrong?

Reply:
I don't want to run `git diff` and all the code below, if option `git-diff` is not used